packages/fxa-settings/src/pages/Signin/SigninBounced/index.test.tsx

```diff
@@ -38,13 +55,33 @@ describe('SigninBounced', () => {
   });
 
   it('renders the "Back" button when a user can go back', () => {
-    renderWithRouter(<SigninBounced email={MOCK_EMAIL} canGoBack={true} />);
+    renderWithRouter(
+      <SigninBounced
+        email={MOCK_ACCOUNT.primaryEmail.email}
+        canGoBack={true}
```

Comment:
nit: can omit `{true}`, although it is a `*.test.*` file. 🤷

Reply:
Fixed!